linux_os/guide/system/network/network-ipv6/disabling_ipv6/kernel_module_ipv6_option_disabled/rule.yml

```
@@ -19,9 +19,12 @@ severity: medium
 
 identifiers:
     cce@rhel6: 27153-6
+    cce@rhel7: 82871-5
+    cce@rhel8: 82872-3
 
 references:
     disa@rhel6: "1551"
+    cis@rhel8: 3.6
```

```suggestion
    cis@rhel8: "3.6"
```

The build system interprets as a float number in this case. It needs to be double quoted

---

Ack & fixed. Thanks!